code/__HELPERS/lists.dm

```diff
@@ -506,4 +506,13 @@
     var/max
     for(var/a in L)
         if(max == null || L[a] > max) max = L[a]
-    return max
\ No newline at end of file
+    return max
+
+//Convert a list of paths into a list of object names
+/proc/types_to_english_list(var/list/L)
+	var/list/names = list()
+	for(var/P in L)
+		if(!ispath(P))
+			continue
+		names += "\the [P:name]"
```

Comment:
Wait, what the fuck? Mysteriously, this actually does work. I don't think it's supposed to, and I wouldn't rely on it continuing to work. It should be `P::name`